src/Compilers/CSharp/Test/Emit2/Semantics/PatternMatchingTests.cs

```diff
@@ -11939,9 +11935,9 @@ public static async Task<bool> ExceptionFilterBroken()
             var comp = CreateCompilation(source, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
             var verifier = CompileAndVerify(comp, expectedOutput: "True");
-            verifier.VerifyIL("C.<ExceptionFilterBroken>d__1.System.Runtime.CompilerServices.IAsyncStateMachine.MoveNext()", @"
- {
-  // Code size      527 (0x20f)
+            verifier.VerifyIL("C.<ExceptionFilterBroken>d__1.System.Runtime.CompilerServices.IAsyncStateMachine.MoveNext()", """
```

Comment:
>""" [](http://example.com/codeflow?start=125&length=3)

Please revert changes like that. Style changes are not welcome. Also, reviewing IL changes is difficult enough to deal with an additional distraction around quotes.

Reply:
VerifyIL outputs raw strings, figured I'd just use IDE refactoring to revert (didn't help much for MoveNext).
The useless diff is not due to raw strings though it's the additional indentation in the base-line.